internal/git/git.go

```diff
@@ -316,6 +319,9 @@ func (g *Git) CommitAndPush(openAPIDocVersion, speakeasyVersion, doc string, act
 	var commitMessage string
 	if action == environment.ActionRunWorkflow {
 		commitMessage = fmt.Sprintf("ci: regenerated with OpenAPI Doc %s, Speakeasy CLI %s", openAPIDocVersion, speakeasyVersion)
+		if sourcesOnly {
+			commitMessage = fmt.Sprintf("ci: regenerated with Speakeasy CLI %s", speakeasyVersion)
```

Comment:
This seems a little unnecessary? Why would we not have openAPIdocversion when doing sources?

Reply:
It comes from the releases file I think the generator gives it to us.

Reply:
<img width="497" alt="Screenshot 2024-03-25 at 12 17 50 PM" src="https://github.com/speakeasy-api/sdk-generation-action/assets/42415738/6affc2d7-61a4-401a-9814-e963d58fbd92">
actually I don't think we get it at all right now